compiler/src/codegen/compcore.re

```diff
@@ -3984,6 +3984,13 @@ let compile_wasm_module = (~env=?, ~name=?, prog) => {
   let () = ignore @@ compile_tables(wasm_mod, env, prog);
   let () = ignore @@ compile_elems(wasm_mod, env, prog);
 
+  let serialized_cmi = Cmi_format.serialize_cmi(prog.signature);
+  Module.add_custom_section(
+    wasm_mod,
+    "cmi",
```

Comment:
We might want to consider renaming this to `grain_cmi` or something in case someone starts putting their CMIs in a custom section named `cmi` 😛

Reply:
Yeah, that's fair. Though at least we sprinkle some [Grain magic](https://github.com/grain-lang/grain/blob/master/compiler/src/utils/wasm_utils.re#L33) to know that it's a Grain CMI. But we could open an issue for that, yeah.